docs/guides/migrating-from-camunda-platform-7/migration-readiness.md

```diff
@@ -0,0 +1,278 @@
+---
+id: migration-readiness
+title: Getting ready to migrate
```

Comment:
Maybe something a little smoother for this title -- Migration preparation or Migration overview?

Reply:
Let's go with "Migration preparation"